### ISSUE TITLE
NTR: Rodent exposure (Immune system assessed clinically)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8047]
New term request Rodent exposure (samant_101210005943): Including prairie dogs  UMLS: 0240913

**Comment by pnrobinson** [comment]
@MickeySegal

should we add this as a Recent past medical history item or as prolonged exposure to rodents?

**Comment by MickeySegal** [comment]
Recent past medical history makes more sense because the diseases (hantavirus, plague, Lassa) have finite incubation periods.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000728
- label: History of recent rodent exposure
- parents: [Recent past medical history (HP:4000144)](https://hpo.jax.org/app/browse/term/HP:4000144)
- pmid: 
- def: A history of exposed to rodents (mice, rats, prarie dogs) within several weeks of the current presenting complaint.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8047
- synonyms:
